skree/README.md

```diff
@@ -0,0 +1,64 @@
+###### Credit for this guide goes to steamport, from [Sunset Station] (https://github.com/SunsetStation/SunsetStation/blob/master/sunset/README.md). Edits done by battletrap360.
+
+# Introduction to the "skree" Folder
+
+## Why do we use it?
+
+To keep up-to-date with /tg/station while trying to avoid merge conflicts as much as possible, we modularize everything. This is the same method [HippieStation](https://github.com/HippieStation/HippieStation/tree/master/hippiestation) uses. The more code in the "skree" folder, the fewer conflicts will exist in future updates.
+
+## What does it mean to modularize something?
+
+Something is modular when it exists independent from the rest of the code. This means that by simply adding something modular to the DME file, it will exist in-game. It is not always possible to completely modularize something, but if standards are followed correctly, then there should be few to none conflicts with /tg/station in the future.
+
+## Please mark your changes
+
+All modifications to non-skree files should be marked.
+
+- Multi line changes start with `// skree start` and end with `// skree end`
+- You can put a messages with a change if it isn't obvious, like this: `// skree start - reason`
+  - Should generally be about the reason the change was made, what it was before, or what the change is
+  - Multi-line messages should start with `// skree start` and use `/* Multi line message here */` for the message itself
+- Single line changes should have `// skree` or `// skree - reason`
+
+If you need to mirror a file, or function into a yog-specific file, please leave behind a comment stating where it went.
+
+```
+// skree start - Mirrored this function in <file> for <reason>
+bunch of shitcode here
+// skree end
+```
+
+Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.
+
+
+### tgstation.dme versus skree.dme
+
+Do not alter the tgstation.dme file. All additions and removals should be to the sunsetstation.dme file. Do not manually add files to the dme! Check the file's box in the Dream Maker program. The Dream Maker does not always use alphabetical order, and manually adding a file can cause it to reorder. This means that down the line, many PRs will contain this reorder when it could have been avoided in the first place.

```

Comment:
sunsetstation.dme?

Reply:
ah, that is a text replacement that I missed